compiler/noirc_frontend/src/ast/mod.rs

```diff
@@ -297,6 +309,15 @@ pub enum Signedness {
     Signed,
 }
 
+impl Signedness {
+    pub fn to_bool(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_signed(&self) -> bool {
```
`to_bool` is too general. What does the bool represent?

Reply:
Done!